### ISSUE TITLE
bug au niveau des entité hostile 

### ISSUE THREAD

**Issue by Ricorsse** [Margouta/PluginOpenMC#320]
pratiquement aucun mob spawn  dan les endroit sombre ou leur nombre est fortement impacté  exemple pour une tour a mob   de 9 étage  on a 1 mob toute les heure

**Comment by Fnafgameur** [comment]
Mob cap.
Ton problème vient de là, une tour à mob n'est pas efficace sur un serveur multi avec beaucoup de joueurs, privilégie les spawners et aménage-les.
De plus, les mobs se stackent donc c'est normal que tu n'en croises pas beaucoup éparpillés.

**Comment by Ricorsse** [comment]
même en étant éloigné de tout et en  pleine océan  couche 200 ? en tout cas ce matin et cette après midi il y avais que  1 mob toute les heure voir pas du tout et la ou écrit le commentaire il y en a quelque un  qui spawn du genre 1 ou 2 toute les 10 min

**Comment by Fnafgameur** [comment]
Yep, même autant éloigné ça ne marche pas très bien, même sur un serveur entre pote avec aucun plugin, si tout le monde est chacun de son côté, ça affectera les perfs de la tour.

**Comment by Ricorsse** [comment]
daccord ce qui serait intéressant  de pouvoir récupérer les spawners grasse à (Silk Touch) par exemple

**Comment by Fnafgameur** [comment]
Pour ça, je ne peux que te conseiller de faire ta proposition dans le channel forum du serveur discord [demandes-serveur](https://discord.com/channels/1161296442577653802/1257444868599447735) pour avoir l'avis de tout le monde

**Comment by sajiterre** [comment]
> daccord ce qui serait intéressant  de pouvoir récupérer les spawners grasse à (Silk Touch) par exemple 

En cours
